actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -85,6 +85,10 @@ module RequestForgeryProtection
       config_accessor :per_form_csrf_tokens
       self.per_form_csrf_tokens = false
 
+      # Controls whether the token should be in a separate persistent cookie or in the session (old behavior).
+      config_accessor :persistent_csrf_token
+      self.persistent_csrf_token = true
+
```

Comment:
i.e. a person wants to go with `null_session` instead of using a separate cookie